nx.json

```diff
@@ -76,23 +76,7 @@
         "command": "eslint src"
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33019&commitId=c4a1f93244b8fd50c691164419a1e5a435ebb7f6&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|29| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|18| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|2818| Changed|
|Drawer.Full Overlay High Contrast.chromium.png|2213| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>